Userland/Libraries/LibGfx/JPEGLoader.cpp

```diff
@@ -201,6 +201,8 @@ struct Scan {
     u8 spectral_selection_start {};
     u8 spectral_selection_end {};
     u8 successive_approximation {};
+
+    HuffmanStreamState huffman_stream;
```

Comment:
"Please note that this patch has intentional implicit behavior. The" -> "Please note that this patch changes behavior. The"